src/libs/actions/IOU.ts

```diff
@@ -6921,13 +6921,13 @@ function duplicateExpenseTransaction(
             ...transaction,
             ...transactionDetails,
             attendees: transactionDetails?.attendees as Attendee[] | undefined,
-            comment: transactionDetails?.comment,
+            comment: Parser.htmlToMarkdown(transactionDetails?.comment ?? ''),
             created: format(new Date(), CONST.DATE.FNS_FORMAT_STRING),
             customUnitRateID: transaction?.comment?.customUnit?.customUnitRateID,
             isTestDrive: transaction?.receipt?.isTestDriveReceipt,
             merchant: transaction?.modifiedMerchant ? transaction.modifiedMerchant : (transaction?.merchant ?? ''),
             modifiedAmount: undefined,
-            originalTransactionID: transaction?.comment?.originalTransactionID,
+            originalTransactionID: undefined,
```

Comment:
Curious if we can completely remove this param?

Reply:
`transaction` and `transactionDetails` don't have `originalTransactionID` so we're safe to remove the params. However, I think we can keep it to avoid unexpected bugs

https://github.com/Expensify/App/blob/e0923e02c9b0c307ea68626879a40a75e9d686c6/src/libs/actions/IOU.ts#L6923-L6924